### ISSUE TITLE
why two optimizers are needed during the training?

### ISSUE THREAD

**Issue by manwu1994** [jayroxis/PINNs#4]
Hello, thanks a lot for your hard work and your sharing.
I would like to ask why two optimizers are needed during the training. 
`self.optimizer.step(self.loss_func)`
`self.optimizer_Adam.step()`

Thank you so much in advance for your answers.

**Comment by jayroxis** [comment]
Hi,

The original PINN uses two optimizers, Adam for initial optimization since it is generally more stable, and later use second order optimizer LBFGS for finetuning to reach higher accuracy.
